Docker/itksoftwareguide-bin/README.md

```diff
@@ -1 +1,21 @@
+Description
+-----------
+
 Holds the volume with the build for the Software Guide.
+
+Volumes
+-------
+
+Source folder: /home/itk/src/ITKSoftwareGuide
+Binary folder: /home/itk/bin/ITKSoftwareGuide
+
+Usage
+-----
+
+Build in the docker container:
+  ./run.sh itksoftwareguide-bin
+
+Use ITKSoftwareGuide outside of the docker container:
+  ./run.sh itksoftwareguide-bin \
+    -v ${HostITKSoftwareGuidSourceDirectory}:/home/itk/src/ITKSoftwareGuide \
+    -v ${HostITKSoftwareGuidBinaryDirectory}:/home/itk/bin/ITKSoftwareGuide
```

Comment:
My google skills are failing me but I swear I once posted what official release had SSL provided for all official binaries.